austation/code/modules/cargo/exports/food_and_drink/processed.dm

```diff
@@ -0,0 +1,54 @@
+//PROCESSED FOOD IS NOW SELLABLE WOOOHOOOO
+//These values are subject to change if it becomes an exploit.
+//Average food price should be around 5cr per step I imagine.
+//UPDATED: 10cr per step
+
+/datum/export/meatball
+	cost = 20
+	unit_name = "meatball"
+	export_types = list(/obj/item/reagent_containers/food/snacks/meatball)
+  
+  /datum/export/raw_bacon
+	cost = 10 		//30 devided by 3 for the 3 cutlets
+	unit_name = "raw bacon slice"
+	export_types = list(/obj/item/reagent_containers/food/snacks/meat/rawbacon)
+  
+  /datum/export/space_fries
+	cost = 30
+	unit_name = "space fries"
+	export_types = list(/obj/item/reagent_containers/food/snacks/fries)
+  
+  /datum/export/yakiimo
+	cost = 20
+	unit_name = "yakiimo"
+	export_types = list(/obj/item/reagent_containers/food/snacks/yakiimo)
+  
+   /datum/export/tatortot
+	cost = 20
+	unit_name = "tatortot"
+	export_types = list(/obj/item/reagent_containers/food/snacks/tatortot)
+  
+   /datum/export/soydope
+	cost = 20
+	unit_name = "soydope"
+	export_types = list(/obj/item/reagent_containers/food/snacks/soydope)
+  
+   /datum/export/spaghetti
+	cost = 13 		//40 divided by 3 dough slices rounded to 13
+	unit_name = "spaghetti"
+	export_types = list(/obj/item/reagent_containers/food/snacks/spaghetti)
+  
+   /datum/export/tortilla
+	cost = 20
+	unit_name = "tortilla"
+	export_types = list(/obj/item/reagent_containers/food/snacks/tortilla)
+  
+     /datum/export/cornchips
+	cost = 30
+	unit_name = "cornchip"
+	export_types = list(/obj/item/reagent_containers/food/snacks/cornchips)
+  
+     /datum/export/roastparsnip
+	cost = 20
+	unit_name = "roast parsnip"
+	export_types = list(/obj/item/reagent_containers/food/snacks/roastparsnip)
```

Comment:
all your shit has the wrong indentation.

Reply:
Ahhhhhh fuck

Reply:
done